src/items/Item.tsx

```diff
@@ -147,12 +152,14 @@ export default class extends React.Component<Props, State> {
         const controllerStyle = {
             display: this.state.hovered ? 'inline' : 'none'
         }
+        const name = this.giveHTMLATag(item.name) || "[NO Name]"
+        const renderName = name === "[NO Name]"? name: <span dangerouslySetInnerHTML={{__html: name }} />
```

Comment:
```suggestion
        const renderName = name === "[No Name]"? name: <span dangerouslySetInnerHTML={{__html: name }} />
```
typo! ;)
サジェスト機能が beta だけど追加されてたので使ってみた